### PR TITLE
Dev/error handling

### PR DESCRIPTION
**Pull request by bakunyo**
- エラー処理機構を導入
  - このあたりを参考にした
  - http://qiita.com/ma2ge/items/938d9f8f4839eb336318
- https://github.com/hr-dash/hr-dash/pull/86 に依存
- エラーページの見た目は後で

### CODE REVIEW

File: spec/requests/monthly_report_comment_spec.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+describe MonthlyReportCommentsController, type: :request do
+  let(:user) { create(:user) }
+  let(:report) { create(:monthly_report) }
+
+  before { login user }
+
+  describe '#create POST /monthly_report_comments' do
+    let(:comment) { build(:monthly_report_comment, monthly_report: report) }
+    let(:comment_params) { { comment: comment.attributes.slice('comment', 'monthly_report_id') } }
```
Comment (freezer530000) [inline spec/requests/monthly_report_comment_spec.rb]: **[rubocop]**
- Line is too long. [98/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

File: spec/requests/monthly_report_comment_spec.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    before { post monthly_report_comments_path, comment_params }
+
+    it { expect(response).to have_http_status :success }
+    it { expect(user.monthly_report_comments.size).to eq 1 }
+    it { expect(user.monthly_report_comments.first.comment).to eq comment.comment }
```
Comment (freezer530000) [inline spec/requests/monthly_report_comment_spec.rb]: **[rubocop]**
- Line is too long. [83/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

File: spec/requests/monthly_report_comment_spec.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  end
+
+  describe '#update PATCH /monthly_report_comments/:id' do
+    let!(:comment) { create(:monthly_report_comment, monthly_report: report, user: user) }
```
Comment (freezer530000) [inline spec/requests/monthly_report_comment_spec.rb]: **[rubocop]**
- Line is too long. [90/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

File: spec/requests/monthly_report_comment_spec.rb [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    let(:after_comment) { build(:monthly_report_comment, monthly_report: report) }
```
Comment (freezer530000) [inline spec/requests/monthly_report_comment_spec.rb]: **[rubocop]**
- Line is too long. [82/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

File: spec/requests/monthly_report_comment_spec.rb [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    let(:comment_params) { { comment: { id: comment.id, monthly_report_id: report.id, comment: after_comment.comment } } }
```
Comment (freezer530000) [inline spec/requests/monthly_report_comment_spec.rb]: **[rubocop]**
- Line is too long. [122/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

File: spec/requests/monthly_report_comment_spec.rb [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+    before { patch monthly_report_comment_path(comment), comment_params }
+
+    it { expect(response).to have_http_status :success }
+    it { expect(user.monthly_report_comments.size).to eq 1 }
+    it { expect(user.monthly_report_comments.first.comment).to eq after_comment.comment }
```
Comment (freezer530000) [inline spec/requests/monthly_report_comment_spec.rb]: **[rubocop]**
- Line is too long. [89/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

File: spec/requests/monthly_report_comment_spec.rb [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+  end
+
+  describe '#destroy DELETE /monthly_report_comments/:id' do
+    let!(:comment) { create(:monthly_report_comment, monthly_report: report, user: user) }
```
Comment (freezer530000) [inline spec/requests/monthly_report_comment_spec.rb]: **[rubocop]**
- Line is too long. [90/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

**Comment by bakunyo** [comment]
development環境でエラーページを確認するには、
`config/environments/development.rb`で

```
config.consider_all_requests_local       = false
```

と設定し、適当にURLの存在しないページにアクセスしたりすれば良い。

**Comment by Pujyoooo** [comment]
レビューします！

**Comment by ta-chibana** [comment]
OKです！！

**Comment by Pujyoooo** [comment]
LGTM

LGTM

**Comment by bakunyo** [comment]
おぉっ
マージできる状態なのにConflict...

**Comment by bakunyo** [comment]
りべーすした

**Pull request merged**